R/idataframe.r

```diff
@@ -27,7 +27,8 @@ idata.frame <- function(df) {
       } else {
         stop("Immutable")
       }
-    }, list(name = name)), envir=self)
+    },
+    list(name = name)), envir=self)
```

Comment:
Probably yes, but lintr doesn't check this currently. Can we leave this for now?